### ISSUE TITLE
Include one-line outreach-oriented note in email footers

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#932]
Currently, emails look something like (depending on if they're questions, comments, notes, etc):

``` markdown
Hi! There's been a new answer posted for the question 'Can we use a timelapse or video camera to monitor particulate air plumes?' that you liked. Do NOT reply to this email; click this link to see the answer:
https://publiclab.org/questions/gretchengehrke/09-20-2016/question-would-it-be-feasible-to-use-a-stationary-camera-to-conduct-emissions-opacity-evaluation#a105

Here is the answer that gretchengehrke wrote:

It turns out that Alternative Method 82 is pretty strict in its procedure, so timelapse and video cameras wouldn't be able to explicitly do that method, but I think they could probably prompt an inspection. Anyway, I recently learned a lot more about EPA Method 9, and some more about Alternative Method 82, and wrote a blog post about it: https://publiclab.org/notes/gretchengehrke/10-28-2016/colorado-regulations-hotsheet

You can comment on the answer at: https://publiclab.org/questions/gretchengehrke/09-20-2016/question-would-it-be-feasible-to-use-a-stationary-camera-to-conduct-emissions-opacity-evaluation#answer-105-comment

Report abuse to: moderators@publiclab.org
```

A typical email (for a new note) is generated here: https://github.com/publiclab/plots2/blob/master/app/views/subscription_mailer/notify_node_creation.html.erb

We could include a line at the bottom using the Features system, using the `feature()` helper:

```
<% cache('feature_email-footer') do %> // <== this caches and uses a cached version of the feature, which is expired when the feature is edited
  <%= feature('email-footer') %>
<% end %>
```

Then the actual text could be set using the features system -- available to admins at `/features`.

**Comment by ykl7** [comment]
@jywarren mind if i take this one?

**Comment by jywarren** [comment]
That would be great, but we may be waiting on a fix from the library that
would generate this...

**Comment by ykl7** [comment]
I could wait for the library fix if needed. If it is not resolved, I'll look into it.

**Comment by jywarren** [comment]
OK -- I'd like to use the library, as manually including names could be
quite onerous every time we publish. Thanks, and if you're looking for a
challenge in the meantime, I've added some new help-wanted issues.

**Comment by ykl7** [comment]
That sounds good

**Comment by ryzokuken** [comment]
@ykl7 Still working on it?

**Comment by ykl7** [comment]
@ryzokuken I was told to wait for a library fix by @jywarren . If that fix doesn't work, I'll do it, but no word yet.

**Comment by ryzokuken** [comment]
okay, cool.

**Comment by jywarren** [comment]
Hi, all -- the issue, in case you want to take a crack at it, is this one: https://github.com/RichardLitt/name-your-contributors/issues/15

If we can get that to work, I think we can move forward with this!

**Comment by jywarren** [comment]
Ack, apologies, I'd mixed up two issues!!! #931 depends on name-your contributors, not this. This is ready to be worked on, @ykl7 -- apologies for the mixup! I'm looking at too many issues these days with all of your excellent contributions coming in so fast! :-)

**Comment by ryzokuken** [comment]
No worries, @jywarren :grin: 

Keep up the good work!

**Comment by ykl7** [comment]
@jywarren I'll try to get this done over the weekend and send a PR as soon as possible

**Comment by ykl7** [comment]
@jywarren Does this have to be added at the bottom of each email sent or will it be just for a new note creation?

**Comment by jywarren** [comment]
I think we could do it for all emails, thanks!

**Comment by jywarren** [comment]
Hello -- how's this going? We could definitely start with just one email, to confirm that it works, then open a new issue to build on your first step. How does that sound?

**Comment by ykl7** [comment]
That sounds good, I was trying to do the same in the last PR I opened, to get one working then work on the rest.

**Comment by jywarren** [comment]
:-) super rad!

**Comment by jywarren** [comment]
Do you think you'd be interested in this somewhat related issue? #931
